### ISSUE TITLE
-Wunneeded-internal-declaration in drivers/gpu/drm/i915/intel_ddi.c

### ISSUE THREAD

**Issue by tpimh** [ClangBuiltLinux/linux#195]
```
drivers/gpu/drm/i915/intel_ddi.c:601:46: warning: variable 'icl_mg_phy_ddi_translations' is not needed and will not be emitted [-Wunneeded-internal-declaration]
static const struct icl_mg_phy_ddi_buf_trans icl_mg_phy_ddi_translations[] = {
                                             ^
1 warning generated.
```

**Comment by nathanchance** [comment]
Should be fixed in -next: https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/commit/?id=07685c827b2a0

**Comment by nathanchance** [comment]
I think that a -next branch was pushed to GitHub at some point because it's not merged on kernel.org: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/log/drivers/gpu/drm/i915/intel_ddi.c

**Comment by tpimh** [comment]
Sorry, my bad. Now I'll be checking on torvalds/linux on kernel.org.

**Comment by nathanchance** [comment]
Now it's been merged! :)

**Comment by nickdesaulniers** [comment]
possibly also fixed on the llvm side by #232
